### DISCUSSION TITLE
Chromedriver tries to update using broken link

### DISCUSSION THREAD

**Discussion by evancampbell** [seleniumbase/SeleniumBase#2737] [Q&A] [answered]
For over a week now, every time I instantiate a Driver with uc=True I get the following:

```
Warning: uc_driver update needed. Getting it now:

*** chromedriver to download = 124.0.6367.91 (Latest Stable) 

Downloading chromedriver-linux64.zip from:
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/124.0.6367.91/linux64/chromedriver-linux64.zip ...
Download Complete!

```

As of posting this, that link is 404'd. I'm able to get around this by using driver_version='keep'.

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/discussions/2665#discussioncomment-9021973

The Chromium Team changed the download location of drivers, and SeleniumBase was updated to use the newer location over two months ago in https://github.com/seleniumbase/SeleniumBase/issues/2519

Upgrade to a newer version of SeleniumBase.
